files/etc/pki/rpm-gpg/RPM-GPG-KEY-elrepo

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
I would prefer to keep the original filename for this GPG Key. Although I don't particularly like the way they named it, I'd rather re-use what they already have in place.  They use ```RPM-GPG-KEY-elrepo.org```